src/coreclr/System.Private.CoreLib/src/System/RuntimeHandles.cs

```diff
@@ -928,7 +967,14 @@ internal static string ConstructInstantiation(IRuntimeMethodInfo method, TypeNam
         }
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        internal static extern RuntimeType GetDeclaringType(RuntimeMethodHandleInternal method);
+        private static extern unsafe MethodTable* GetMethodTable(RuntimeMethodHandleInternal method);
```

Comment:
Looks like this type has an unused property https://github.com/dotnet/runtime/blob/c5235a64cf7a8e9448a6fe02df30c7c79607cbdd/src/coreclr/System.Private.CoreLib/src/System/RuntimeHandles.cs#L785

also maybe better to have one m_handle accessor visible:
```diff
        internal IntPtr Value => m_handle;
...
-        internal IntPtr m_handle;
+        private IntPtr m_handle;
```